friktion/friktion/bid_details.py

```diff
@@ -1,38 +1,17 @@
+from dataclasses import dataclass
+
 from anchorpy import Wallet
 from nacl import signing  # type: ignore
 from solana.publickey import PublicKey
 
-from friktion.pda import SwapOrderAddresses
-
 
+@dataclass
 class BidDetails:
```

Comment:
Trying to reduce the parameters to only to the ones used when signing (as_signed_msg)

Reply:
looks good to me